src/Compilers/CSharp/Test/Emit2/Semantics/RefReadonlyParameterTests.cs

```diff
@@ -0,0 +1,635 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Semantics
```

Comment:
Consider using file-scoped namespace (we tend to do that for new test files)

Reply:
> The main reason I advocate this is because test files tend to have long lines (such as diagnostics).

I dislike this form of namespaces no matter what.